oshc/oshc/settings.py

```diff
@@ -27,9 +26,9 @@
 
 # Tuple of people who get error notifications
 ADMINS = [
-    ('Tapasweni Pathak','tapaswenipathak@gmail.com'),
-    ('Nikhita Raghunath','nikitaraghunath@gmail.com'),
-    ('Ibrahim Jarif','jarifibrahim@gmail.com'),
+    ('Tapasweni Pathak', 'tapaswenipathak@gmail.com'),
+    ('Nikhita Raghunath', 'nikitaraghunath@gmail.com'),
+    ('Ibrahim Jarif', 'jarifibrahim@gmail.com'),
```

Comment:
@vaibhavsingh97  Please remove these spaces

Reply:
@anubhakushwaha According to PEP 8, we should have spaces. If you want i can remove.

Reply:
@vaibhavsingh97 Ooh that's interesting, we would love to learn more,Is this the place? https://www.python.org/dev/peps/pep-0008/#tabs-or-spaces

Reply:
@anubhakushwaha Refer here: https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements